tbx/static_src/sass/config/_variables.scss

```diff
@@ -184,6 +186,7 @@ $color--earth-streamfield-background: #e6e3df;
     --color--text-50: #{color.adjust($color--grey-70, $alpha: -0.5)};
     --color--decoration: #{$color--eclipse};
     --color--link: #{$color--eclipse};
+    --color--link-underline: #{$color--eclipse};
```

Comment:
Maybe double check with ben what this colour should be - we might want it to be a bit lighter in comparison to the text colour.

Reply:
I've asked him on the ticket